src/mixins/itext_key_behavior.mixin.js

```diff
@@ -239,7 +239,9 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
     }
 
     fabric.copiedText = this.getSelectedText();
-    fabric.copiedTextStyle = this.getSelectionStyles(this.selectionStart, this.selectionEnd, true);
+    if (!this.disableStyleCopying) {
+      fabric.copiedTextStyle = this.getSelectionStyles(this.selectionStart, this.selectionEnd, true);
+    }
```

Comment:
we need to assign something to this object anyway.
Imagine someone copy with style.
Then enable the property.
Then copy again. The old copied style is still there.
I think an else statement is ok or just assign it to null ( whatever is the default )  before the if.

Reply:
I guess I had been thinking of this more like an application level setting, in which case the object would remain empty regardless. If you see this as a property that some users might want to set differently for different text objects then I wonder if this approach is going to be problematic since it won't prevent a box that's meant to contain only one style from still receiving style blocks from text with this option turned off. Would it perhaps be better to add a `disableStylePasting` property instead? Or maybe both?

Reply:
the property is on `fabric` so is at webpage level, not even application.
The point is that then devs do what they want to do, that is a property that you can enable or disable and people will do in the way we did not think of.

So better it behaves consistently

Reply:
Well we can call it `disableStyleCopyPaste` and can stop copy and pasting all together.

Reply:
Wait ok, i see you added it on `this` i would expect it to be fabric.disableStyleCopyPasting. Per object does not make sense in my opinion.

Reply:
Right, yes attaching the property to fabric does make more sense (I can't imagine why anyone would need it to be per object)